stable/openvpn/Chart.yaml

```diff
@@ -3,7 +3,7 @@ description: A Helm chart to install an openvpn server inside a kubernetes clust
   generation is also part of the deployment, and this chart will generate client keys
   as needed.
 name: openvpn
-version: 2.0.4
+version: 3.0.4
```

Comment:
If you increment to version 3 it should be `3.0.0`. Can you fix that?

Reply:
Yep! Pushed a fix.